renku/ui/service/views/error_handlers.py

```diff
@@ -375,8 +378,26 @@ def decorated_function(*args, **kwargs):
     return decorated_function
 
 
-def handle_migration_errors(f):
-    """Wrapper which handles migrations exceptions."""
+def handle_migration_read_errors(f):
+    """Wrapper which handles migrations read exceptions."""
+    # noqa
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        """Represents decorated function."""
+        # NOTE: verify if this may better go in MigrationsCheckCtrl as try/except in to_response()
```

Comment:
I don't get this comment. Is it a left-over TODO?

Reply:
Ouch! Yes, that is a leftover, thanks for spotting it!